scaleway/resource_lb_subscriber_beta.go

```diff
@@ -0,0 +1,146 @@
+package scaleway
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/scaleway/scaleway-sdk-go/api/lb/v1"
+)
+
+func resourceScalewayLbSubscriberBeta() *schema.Resource {
+	return &schema.Resource{
+		Create:        resourceScalewayLbSubscriberBetaCreate,
+		Read:          resourceScalewayLbSubscriberBetaRead,
+		Update:        resourceScalewayLbSubscriberBetaUpdate,
+		Delete:        resourceScalewayLbSubscriberBetaDelete,
+		SchemaVersion: 0,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Description: "Subscriber name.",
+				Optional:    true,
+				Computed:    true,
+			},
+			"email_config": {
+				ConflictsWith: []string{"webhook_config"},
+				MaxItems:      1,
+				Description:   "Email address configuration. Only one of email_config and webhook_config may be set",
+				Type:          schema.TypeList,
+				Optional:      true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"email": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "Email who receive alert.",
+						},
+					},
+				},
+			},
+			"webhook_config": {
+				ConflictsWith: []string{"email_config"},
+				MaxItems:      1,
+				Description:   "WebHook URI configuration. Only one of email_config and webhook_config may be set.",
+				Type:          schema.TypeList,
+				Optional:      true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"uri": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "URI who receive POST request.",
```

Comment:
URI receives the POST request.
```suggestion
							Description: "URI receiving the POST request.",
```